pyanaconda/ui/tui/spokes/installation_source.py

```diff
@@ -426,7 +432,8 @@ def refresh(self, args=None):
         # check if there are any mountable devices
         if self._mountable_devices:
             for d in self._mountable_devices:
-                self._container.add(TextWidget(d[1]), callback=self._select_mountable_device, data=d[0])
+                self._container.add(TextWidget(d[1]), callback=self._select_mountable_device,
+                                    data=d[0])
```

Comment:
Personally, I don't like this style. I prefer a new line for every argument.

Reply:
Fair, I'll change that.